docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -152,6 +152,10 @@ described in [Teleport Upcoming Releases](../../upcoming-releases.mdx).
 - Teleport Auth Service and Teleport Proxy Service updates are pushed to Teleport cloud accounts
   during your scheduled maintenance window.
 - Teleport agents are only updated automatically if you enroll them in automatic updates.
+- Teleport Auth and Proxy service will not be automatically upgraded to the next major version with
+  Teleport agents on older major versions. An example would be the Teleport Auth and Proxy service are on
+  version 16 and agents are still on version 15 or older. That would prevent Teleport Auth and Proxy
+  being updated to version 17 automatically.
```

Comment:
I would reorder these bullet points. As written, we discuss auth/proxy, then we discuss agents, then we go back to discussing auth and proxy.